spec/src/modules/tracker.js

```diff
@@ -525,7 +525,7 @@ describe('ConstructorIO - Tracker', () => {
         fetch: fetchSpy,
       });
 
-      expect(tracker.trackConversion(term, parameters)).to.equal(true);
+      expect(tracker.trackConversion(term, {})).to.equal(true);
```

Comment:
We were passing `section` as part of the parameters, which defeats the point of the test as we're testing to see that `section` is defaulted when not supplied.